src/devices/Interop/Unix/Libasound/Interop.libasound.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [DllImport(AlsaLibrary)]
+    internal static extern IntPtr snd_strerror(int errnum);
```

Comment:
Have you perhaps had a chance to test this on ARM64? IntPtr size depends on the architecture so I'd feel better with this if we did

Reply:
fine to just add a comment in the readme that this wasn't tested on 64-bit ARM and to investigate interop types if there are any issue